crates/bitwarden-core/Cargo.toml

```diff
@@ -39,6 +39,7 @@ chrono = { version = ">=0.4.26, <0.5", features = [
     "std",
 ], default-features = false }
 coset = { version = "0.3.7", optional = true }
+dirs = "5.0.1"
```

Comment:
Hmm, not a huge fan of adding this dependency to the bitwarden-core crate. It also feels weird for a library to have side effects and write files without being prompted.